### PR TITLE
NOISSUE - Fix Readers logs

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

File: readers/cassandra/messages_test.go [hunk 1]
```diff
@@ -147,7 +147,10 @@ func TestReadAll(t *testing.T) {
 	}
 
 	for desc, tc := range cases {
-		result := reader.ReadAll(tc.chanID, tc.offset, tc.limit, tc.query)
+		result, err := reader.ReadAll(tc.chanID, tc.offset, tc.limit, tc.query)
+		if err != nil {
+			assert.Equal(t, nil, err, fmt.Sprintf("%s: expected no error got %s", desc, err.Error()))
+		}
```
Comment (anovakovic01) [inline readers/cassandra/messages_test.go]: Don't use `if err != nil {`. Use `assert.Nil` instead of this `if` and `assert.Equal`.

**Review by drasko — DISMISSED** [review]
LGTM

File: readers/influxdb/messages_test.go [hunk 2]
```diff
@@ -166,7 +166,10 @@ func TestReadAll(t *testing.T) {
 	}
 
 	for desc, tc := range cases {
-		result := reader.ReadAll(tc.chanID, tc.offset, tc.limit, tc.query)
+		result, err := reader.ReadAll(tc.chanID, tc.offset, tc.limit, tc.query)
+		if err != nil {
+			assert.Equal(t, nil, err, fmt.Sprintf("%s: expected no error got %s", desc, err.Error()))
+		}
```
Comment (anovakovic01) [inline readers/influxdb/messages_test.go]: Same here, use `assert.Nil` instead.

File: readers/mongodb/messages_test.go [hunk 3]
```diff
@@ -155,7 +155,10 @@ func TestReadAll(t *testing.T) {
 	}
 
 	for desc, tc := range cases {
-		result := reader.ReadAll(tc.chanID, tc.offset, tc.limit, tc.query)
+		result, err := reader.ReadAll(tc.chanID, tc.offset, tc.limit, tc.query)
+		if err != nil {
+			assert.Equal(t, nil, err, fmt.Sprintf("%s: expected no error got %s", desc, err.Error()))
+		}
```
Comment (anovakovic01) [inline readers/mongodb/messages_test.go]: Same here, replace with `assert.Nil`.

File: readers/cassandra/messages.go [hunk 4]
```diff
@@ -85,21 +87,21 @@ func (cr cassandraRepository) ReadAll(chanID string, offset, limit uint64, query
 	}
 
 	if err := iter.Close(); err != nil {
```
Comment (anovakovic01) [inline readers/cassandra/messages.go]: Please ignore this error and move `iter.Close()` to line 44 of this file. So it should be `defer iter.Close()`.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

File: readers/cassandra/messages_test.go (hunk 1 above)
Comment (manuio) [reply readers/cassandra/messages_test.go]: Thanks!

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**